index.js

```diff
@@ -14,7 +14,8 @@ module.exports = function (ms) {
 		});
 	}
 
-	thunk.then = promise.then.bind(promise);
+        thunk.then = promise.then.bind(promise);
+        thunk.catch = promise.catch.bind(promise);
```

Comment:
Fixed. I usually use webstorm, which has EditorConfig, but I was being lazy and trying to do it in vim, so as usual, it went horrifically wrong.